<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flight_search</class>
 <widget class="QWidget" name="flight_search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>航班搜索</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#flight_search{background-image: url(:/Image/Imageres/Bookings.jpg);}</string>
  </property>
  <widget class="QLabel" name="labelBackground">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-60</y>
     <width>900</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
background-image: url(:/Imageres/airBg.jpeg);</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>航班搜索</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ReturnButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#ReturnButton{
	image: url(:/Image/Imageres/back.png);
    background: transparent; /* 背景透明 */
    border: none; /* 移除边框 */
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>832</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelTakeoff">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>出发地</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditChu">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid gray; border-radius: 5px; padding: 2px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelArrival">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>到达地</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditDao">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid gray; border-radius: 5px; padding: 2px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDate">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>日期</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditDate">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid gray; border-radius: 5px; padding: 2px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border-radius:4px;background:#409eff;color:white;font-size:14px}
QPushButton:hover {
         background-color: #ADD8E6;&quot;        /* 悬停时背景颜色 */
        }</string>
        </property>
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="styleSheet">
       <string notr="true">background-color: white; border: 1px solid gray; border-radius: 5px;</string>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>9</number>
      </property>
      <column>
       <property name="text">
        <string>航班号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>经济舱票价</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>超级经济舱票价</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>头等舱票价</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>经济舱人数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>超级经济舱人数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>头等舱人数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>操作</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
